src/main/java/org/fogbowcloud/manager/utils/PropertiesUtil.java

```diff
@@ -0,0 +1,27 @@
+package org.fogbowcloud.manager.utils;
+
+import java.util.Properties;
+
+import org.fogbowcloud.manager.core.SharedOrderHolders;
+import org.fogbowcloud.manager.core.manager.constants.ConfigurationConstants;
+
+public class PropertiesUtil {
```

Comment:
This class seems useless to me, do you want to map some common properties in tests?